config/detekt/baseline.xml

```diff
@@ -20,622 +20,1461 @@
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun provideObserveServerConfigUseCase(@KaliumCoreLogic coreLogic: CoreLogic)</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun provideObserveSyncStateUseCase(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId)</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun providePersistOtherUsersClients(@CurrentAccount currentAccount: UserId, clientScopeProviderFactory: ClientScopeProvider.Factory)</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun providePersistPersistentWebSocketConnectionStatusUseCase( @KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId )</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun providePersistReadReceiptsStatusConfig(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId)</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun provideRejectCallUseCase(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId)</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun provideRenameConversation(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId)</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun provideRequestVideoStreamsUseCase(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId)</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun provideRestartSlowSyncProcessForRecoveryUseCase( @CurrentAccount currentAccount: UserId, clientScopeProviderFactory: ClientScopeProvider.Factory )</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun provideRestoreBackupUseCase(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId)</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun provideSelfClientsUseCase(@CurrentAccount currentAccount: UserId, clientScopeProviderFactory: ClientScopeProvider.Factory)</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun provideSetUserHandleUseCase(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId)</ID>
+        <ID>LibraryCodeMustSpecifyReturnType:CoreLogicModule.kt$UseCaseModule$@ViewModelScoped @Provides fun provideUpdateApiVersionsScheduler(@KaliumCoreLogic coreLogic: CoreLogic)</ID>
```

Comment:
This is weird....

@yamilmedina any guesses on why `LibraryCodeMustSpecifyReturnType` is being tracked by Detekt on Reloaded?